test/mpi/distributed/matrix.cpp

```diff
@@ -549,25 +527,21 @@ class HostToDeviceLogger : public gko::log::Logger {
 };
 
 
-class MatrixGpuAwareCheck : public ::testing::Test {
+class MatrixGpuAwareCheck : public CommonMpiTestFixture {
 public:
-    using value_type = double;
     using local_index_type = gko::int32;
     using global_index_type = gko::int64;
     using part_type =
         gko::distributed::Partition<local_index_type, global_index_type>;
-    using dist_mtx_type = gko::distributed::Matrix<value_type, local_index_type,
-                                                   global_index_type>;
-    using dist_vec_type = gko::distributed::Vector<value_type>;
+    using dist_mtx_type =
+        gko::experimental::distributed::Matrix<value_type, local_index_type,
+                                               global_index_type>;
+    using dist_vec_type = gko::experimental::distributed::Vector<value_type>;
     using dense_vec_type = gko::matrix::Dense<value_type>;
 
     MatrixGpuAwareCheck()
-        : ref(gko::ReferenceExecutor::create()),
-          comm(MPI_COMM_WORLD),
-          logger(gko::share(HostToDeviceLogger::create())),
-          engine()
+        : logger(gko::share(HostToDeviceLogger::create())), engine()
```

Comment:
```suggestion
        : logger(gko::share(HostToDeviceLogger::create())), engine(42)
```
it is unnecessary. it's just from other engines use seed